Orange/widgets/unsupervised/owkmeans.py

```diff
@@ -133,18 +146,20 @@ class Warning(widget.OWWidget.Warning):
         )
         no_sparse_normalization = widget.Msg("Sparse data cannot be normalized")
 
-    INIT_METHODS = (("Initialize with KMeans++", "k-means++"),
-                    ("Random initialization", "random"))
+    BASE_INIT_METHODS = (("Initialize with KMeans++", "k-means++"),
+                         ("Random initialization", "random"))
+    DASK_INIT_METHODS = (("Initialize with KMeans||", "k-means||"),)  # dask_ml default (parallelized k-means++))
 
     resizing_enabled = False
 
+    settingsHandler = TableTypeContextHandler()
     k = Setting(3)
     k_from = Setting(2)
     k_to = Setting(8)
     optimize_k = Setting(False)
     max_iterations = Setting(300)
     n_init = Setting(10)
-    smart_init = Setting(0)  # KMeans++
```

Comment:
This change, if we go with it, also requires settings migration. Just leaving it here as a note.